recipe/meta.yaml

```
@@ -24,17 +24,34 @@ requirements:
     - python
     - pydantic >=1.0.0,<3.0.0
     - requests >=2.0.0,<3.0.0
+    - orjson >=3.9.14,<4.0.0
 
 test:
+  source_files:
+    - python/tests
+    - python/pyproject.toml
   imports:
     - langsmith
     - langsmith.schemas
     - langsmith.client
   requires:
     - pip
+    - pytest
+    - pytest-asyncio
+    - pytest-rerunfailures
+    - attrs
+    - dataclasses-json  # [py<312]
```

```suggestion
    - dataclasses-json
```
I opened a PR to get a 3.12 build of dataclasses-json
https://github.com/AnacondaRecipes/dataclasses-json-feedstock/pull/4

---

Thanks!